device_backends/LogicalNameMapping/src/LNMMathPlugin.cc

```diff
@@ -361,13 +286,23 @@ namespace ChimeraTK::LNMBackend {
     if(!_p->_isWrite) {
       throw ChimeraTK::logic_error("This register with MathPlugin enabled is not writeable: " + _target->getName());
     }
+    // LNM backend - we know it exists
+    auto backend = h->_backend.lock();
+    if(!backend->isOpen()) {
```

Comment:
According to the [TransferElement spec Section C.2](https://chimeratk.github.io/DeviceAccess/master/spec__transfer_element.html#transferElement_C_2), we are not allowed to throw any new exception in the post phase. This may cause issues with other decorators (e.g. ExceptionHandlingDecorator in ApplicationCore).

Reply:
> If I am not mistaken doPostxxx must never throw logic errors according to spec. We might have missed this because we never thought about close(). Probably the right behaviour (according to current spec) would be to throw in close() if there are still pending post-reads(). This might not even be implementable. I guess we have to extend the spec.

You are just commenting a code line for doPreWrite.   MathPluginDecorator<UserType>::doPostWrite indeed does not throw new exceptions, so I think we have no problem here.